src/components/NFTMedia/NFTMedia.module.scss

```diff
@@ -6,13 +6,15 @@
 	height: 100%;
 	width: 100%;
 	overflow: hidden;
+	border-radius: 8px 8px 0 0;
```

Comment:
`NFTMedia` is used in a few different places, some of which don't have `border-radius` (`NFTView` for example). Can we set `border-radius` in the component rendering the `NFTMedia` by passing a `className` (or similar)?

Reply:
Hmm I checked the images again and looked as though these radius' weren't required. I've just removed these, going to let it build a preview then check out on mobile.